source/js/components/distance-filter-status.js

```diff
@@ -0,0 +1,20 @@
+const updateDistanceFilterStatus = (settings) => {
+
+    console.log('called');
+
+    const filterActiveDistanceEl = document.querySelector('.js-active-distance');
```

Comment:
[fix] You are re-querying the dom for these elements every time this function is run.